documentation/API_documentation/network-access-management-API-Readiness-Checklist.md

```diff
@@ -0,0 +1,29 @@
+# API Readiness Checklist
+
+Checklist for network-access-management 0.1.0-alpha.1 in r1.0-rc.1.
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:---------------:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y   | [relative link](../../code/API_definitions/network-access-management.yaml)  |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |   Y   | 0.5.0          |
```

Comment:
Commonalities is now at r2.3

Reply:
I am a little confused as to what goes in the checklist for commonalities.  ICM at r2.3 makes sense.  For commonalities, the version in the artifacts is 0.6.0, but it is 0.5.0 for the x-header and root version, while the tag now seems to track ICM with r2.3.

https://github.com/camaraproject/Commonalities/blob/cbf7c4fe0442e845fc50d8f887c9c4a9b7963a18/artifacts/CAMARA_common.yaml#L8-L9

```md
  version: 0.6.0
  x-camara-commonalities: 0.5
```

https://github.com/camaraproject/Commonalities/blob/cbf7c4fe0442e845fc50d8f887c9c4a9b7963a18/VERSION.yaml#L1

```md
version: 0.5.0
```

https://github.com/camaraproject/Commonalities/releases/tag/r2.3

Reply:
All other changes have been made including updating the artifacts.

Reply:
Update made to r2.3